### PR TITLE
new Wasteplanet Ruin [The Ship Graveyard]

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request

This PR adds a new wasteplanet ruin, The Ship Graveyard!
--

---

it's an old Scrapyard, which was abandoned. A lot of the Old yardcrew were left behind, and are now fighting within the old hulking wrecks. Will you be the one to take control over it, and extract the old treasures?

---

LOOT:
1x AKSU
1x M90
2x polymer rifle
1x winchester rifle
1x sabre
1x sci imprinter
1x commander pistol
a lot of misc scrap
misc minor loot
a Tattered Centcom Formal Coat

## Why It's Good For The Game

we NEED new wasteplanet ruins

## Changelog

:cl:
add: Added a new wasteplanet ruin (ship_graveyard)
/:cl:

### CODE REVIEW

**Comment by spockye** [comment]
don't you dare fail

**Comment by meemofcourse** [comment]
*fails ur checks

**Review by goober3 — CHANGES_REQUESTED** [review]
2107328b7d2dc7d6c5d543950e5cf05b
607a84ecdf4180b29892e87e9fa4334b
Unrotated Airlocks


image
d4123ffdee219c1f6a44134932ec10d9
7fa6d614b7c5d1e636da5bcccfc8e448
4542687d01fdd6f0dccb216bd0f2002e
Stacked Wallmounts


b4528a373fbb40feed293ec58dd7a152
Floating Extinguisher Cabinet


7ad536e94ef489945a8c7695d82d1c2e
Floating Light


c3c74eae8f25acfb3ed42f27b8d71442
Stray Pipes. it would be so real if you fully atmos'd these ships.

6be3fcf9190d3fedf6816d0b8f1bb15a
add corners to these warning stripes, please!!

8ccf249e7b776af33ce6c16988950317
This Seems Excessive! 

4753a800447b7a8aaf6949f1a33e62af
tone down the loot in this room, please! one pair of magboots should do fine.

35f670b03ad610307d97fd05b93085e0
the elusive Wall Mattress.

File: code/game/area/areas/ruins/wasteplanet.dm [hunk 1]
```diff
@@ -43,3 +43,19 @@
...
+/area/ruin/wasteplanet/wasteplanet_graveyard //for the scrapping post.
+	name = "Abandoned Scrapping Post"
+	icon_state = "disposal"
+
+/area/ruin/wasteplanet/wasteplanet_graveyard/nanotrasen_ship //old nanotrasen carrier, dont mind this
+	name = "Scrapped Nanotrasen Carrier"
+	icon_state = "blue"
+
+/area/ruin/wasteplanet/wasteplanet_graveyard/syndicate_ship //ruined cargo hauler, dont mind this
+	name = "Scrapped Syndicate Haluer"
```
Comment (goober3) [inline code/game/area/areas/ruins/wasteplanet.dm]: haluer

**Review by goober3 — CHANGES_REQUESTED** [review]

File: _maps/map_catalogue.txt [hunk 2]
```diff
@@ -609,3 +609,7 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
 		File name = "_maps\RandomRuins\wasteruins\wasteplanet_abandoned_mechbay
 		Size = (x = 45)(y = 47)(z = 1)
 		Tags = "Boss Combat Challenge", "Medium Loot", "Shelter", "Hazardous"
+
+		File name = "_maps\RandomRuins\wasteruins\wasteplanet_ship_graveyard"
+		Size = (x = 102)(y = 75)(z = 1)
+		Tags = "Medium Loot", "Shelter", "Hazardous"
```
Comment (Latentish) [inline _maps/map_catalogue.txt]: You need to add a combat challenge tag.

**Review by Latentish — DISMISSED** [review]

File: code/game/area/areas/ruins/wasteplanet.dm (hunk 1 above)
Comment (spockye) [reply code/game/area/areas/ruins/wasteplanet.dm]: :c

**Review by goober3 — CHANGES_REQUESTED** [review]
7f45db3d39ca3eea8d0ffe1655c8513b
3bcc7aef70de6801ccdc89138c83843b
Oops ! More Extinguishers. (patch up the turf passthrough, also)

**Review by goober3 — APPROVED** [review]

**Review by Erikafox — CHANGES_REQUESTED** [review]
image
just use one iron ore and varedit the amount var

image
Add some scattered cable coils / scrap, show it was Being Disassembled

image
Why is this here (outside the scrap yard), why is it completely intact. Why is it packed with loot?
Bad wallmount on the newscaster.
image
Use some frontiersmen tbh
image
Good god no. 

Overall needs another pass to make it scrappier + consider introducing my favorite things: Hivebots, they literally scavenge and break stuff down so they'd fit well here.

**Pull request closed without merging**